etc/orthography.tsv

```diff
@@ -807,3 +807,189 @@ řː	rː
 âː	aː
 ́d	d
 ɔ̂ː	ɔː
+iːɐ̯	iːɐ
+oi̯	oi
+œy̯	œy
+aʊ̯	aʊ
+ɔu̯	ɔu
+ʊ̯ɔ	ʊɔ
+aɪ̯	aɪ
+ie̯	ie
+aɐ̯	aɐ
+iɔ̯	iɔ
+oɐ̯	oɐ
+œɐ̯	œɐ
+i̯a	ia
+ɔʊ̯	ɔʊ
+o̯a	oa
+ɛʊ̯ˑ	ɛʊˑ
+ʊe̯	ʊe
+u̯ɪ	uɪ
+ai̯ˀ	ai ʔ
+œu̯ˀ	œu ʔ
+ɔːi̯	ɔːi
+œi̯	œu
+ɪɐ̯	ɪɐ
+ɔu̯ˀ	ɔa ʔ
+uˑi̯	uˑi
+yːɐ̯	yːɐ
+æi̯	æi
+ɛu̯	ɛu
+e̯a	ea
+ɔ̯ɑː	ɔɑː
+ɑi̯ˀ	ɑi ʔ
+ɑʊ̯ˈ	ɑʊ
+aːe̯	aːe
+u̯æ	uæ
+ɒʊ̯ˑ	ɒʊˑ
+ʊʲɪ̯ˑ	ʊɪˑ
+iu̯	iu
+uːɐ̯	uːɐ
+ɐʲɪ̯ˑ	ɐɪˑ
+aːɐ̯	aːɐ
+eø̯	eø
+æe̯	æe
+e̯æ	eæ
+ʊːi̯	ʊːi
+ɑu̯	ɑu
+əo̯	əo
+oʊ̯	oʊ
+aːi̯	aːi
+ɑɪ̯	ɑɪ
+iˑu̯	iˑu
+ɐ̯ɐ	ɐɐ
+iɐ̯ˀ	iɐ ʔ
+iɛ̯	iɛ
+ɑɐ̯	ɑɐ
+uɔ̯	uɔ
+ɔ̯ɛː	ɔɛː
+ou̯	ou
+e̯o	eo
+yˑu̯	yˑu
+ɛʲɪ̯ˑ	ɛɪˑ
+eo̯	eo
+ɑi̯	ɑi
+yœ̯	yœ
+i̯æ	iæ
+ɛɐ̯	ɛɐ
+ɔ̯aː	ɔaː
+ɛi̯	ɛi
+ei̯	ei
+ɔʏ̯	ɔʏ̯
+ɑɔ̯	ɑɔ
+aɛ̯	aɛ
+ɑe̯	ɑe
+uːi̯	uːi
+aːɔ̯	aːɔ
+ɛːɐ̯	ɛːɐ
+u̯ɛ	uɛ
+aːu̯	aːu
+ae̯	ae
+ɔɐ̯	ɔɐ
+ɒɐ̯	ɒɐ
+oe̯	oe
+ʊi̯	ʊi
+ui̯	ui
+ai̯	ai
+əɐ̯	əɐ
+i̯e	ie
+eːɐ̯	eːɐ
+ɔ̯ɑ	ɔɑ
+eːu̯	eːu
+au̯	au
+eu̯	eu
+u̯o	uo
+ɔi̯	ɔi
+ʏɐ̯	ʏɐ
+ɐu̯	ɐu
+ao̯	ao
+oːɐ̯	oːɐ
+ʊɐ̯	ʊɐ
+oːi̯	oːi
+ɔe̯	ɔe
+ɪaˑ	ɪaˑ
+oa	oa
+ɛʊ̃	ɛʊ̃
+ɔã	ɔã
+aiː	aiː
+ɔɛ̃	ɔɛ̃
+ɪuː	ɪuː
+eɑ̃	eɑ̃
+ɔa	ɔa
+ɐu	ɐu
+ɐuː	ɐuː
+ɐuˑ	ɐuˑ
+ɤi	ɤi
+ɛoː	ɛoː
+ʊa	ʊa
+əũ	əũ
+iɛ̃	iɛ̃
+ao̞	ao
+ɪẽː	ɪẽː
+uy	uy
+ɛi	ɛi
+aˑʊ	aˑʊ
+ɪoː	ɪoː
+ua	ua
+əɪ	əɪ
+ɛɔ	ɛɔ
+ɪa	ɪa
+ʊy	ʊy
+ɔãː	ɔãː
+ɪu	ɪu
+ɛɤ	ɛɤ
+ʉaː	ʉaː
+o̞a	oa
+ɛa	ɛa
+o̞ːõ̞	oː
+eːã	eːã
+ei	ei
+yiː	yiː
+uaː	uaː
+ea	ea
+oi	oi
+ae̞	ae
+ɐi	ɐi
+ueː	ueː
+ɪʊ̃	ɪʊ̃
+ao	ao
+ʊãː	ʊãː
+iɔ̃	iɔ̃
+ɨu	ɨu
+iːã	iːã
+aɯ	aɯ
+yˑu	yˑu
+ɨi	ɨi
+uə	uə
+ou	ou
+aæ	aæ
+uɨ	uɨ
+au	au
+ɔɛ̃ː	ɔɛ̃ː
+ɑʊ	ɑʊ
+ye	ye
+iaː	iaː
+ɔu	ɔu
+əaː	əaː
+eẽ	eẽ
+ɨə	ɨə
+ɪã	ɪã
+ʊi	ʊi
+ʉɛ	ʉɛ
+iy	iy
+aeː	aeː
+auː	auː
+ɔ̯ɑ̃ː	ɔɑ̃ː
+aõ̯ 	aõ
+əɪ̆	əɪ̆
+ɛaː	ɛaː
+ɛĩ̯ 	ɛĩ
+ɛɔ̃	ɛɔ̃
+uə̃	uə̃
+ɪaː	ɪaː
+ʊaː	ʊaː
+aoː	aoː
+ɪa͡ɪ	ɪ a͡ɪ
```

Comment:
note that the CLTS system does not allow the bar, it will anyway remove it. So if you encounter this, just write aɪ instead of a͡ɪ. You will see this if you use the `check_profile` command I mentioned before, where it lists all the "modified" characters, which are directly adjusted by CLTS.

Reply:
These are the modified phonemes after removing the markers: https://pastebin.com/5CZ9ahqL